pubsub/google/cloud/pubsub/connection.py

```diff
@@ -104,7 +106,8 @@ class _PublisherAPI(object):
     def __init__(self, connection):
         self._connection = connection
 
-    def list_topics(self, project, page_size=None, page_token=None):
+    @staticmethod
+    def list_topics(project, page_size=None, page_token=None):
```

Comment:
If the API instance had a `self._client` attribute, this would go back to being a "normal" method.